src/importexport/musicxml/internal/musicxml/import/importmusicxmlpass2.cpp

```diff
@@ -6622,6 +6622,23 @@ Note* MusicXmlParserPass2::note(const String& partId,
             } else {
                 headGroup = convertNotehead(noteheadValue);
             }
+        } else if (m_e.name() == "notehead-text") {
+            String noteheadText;
+            while (m_e.readNextStartElement()) {
+                if (m_e.name() == "display-text") {
+                    noteheadText = m_e.readText();
+                } else if (m_e.name() == "accidental-text") {
+                    m_e.skipCurrentElement();
+                } else {
+                    skipLogCurrElem();
+                }
+            }
+            if (noteheadText.size() == 1) {
+                headScheme = (noteheadText == u"H")
+                             ? NoteHeadScheme::HEAD_PITCHNAME_GERMAN : NoteHeadScheme::HEAD_PITCHNAME;
+            } else {
+                headScheme = NoteHeadScheme::HEAD_SOLFEGE_FIXED;
```

Comment:
I suppose we could use some more regex to determine if this should be solfege or just plain note names, but I'm happy with this as is.